docker-compose.yml

```diff
@@ -11,14 +11,8 @@ services:
     depends_on:
       - redis
 
-    environment:
-      - BOT_TOKEN
-      - BOT_DEBUG
-      - BOT_GUILD
-      - BOT_ADMIN_ROLE_ID
-      - CHANNEL_DEVLOG
-      - CHANNEL_COMMUNITY_BOT_COMMANDS
-      - REDIS_HOST=redis
```

Comment:
If this doesn't already have a default value set in Python then it should. Alternatively, the default can remain in the compose file but I prefer defining it in Python.

Reply:
The default is currently set to `redis.default.svc.cluster.local` https://github.com/python-discord/sir-lancebot/blob/c6353a8613d8e30d66a91fc4ed4017b16c4390ab/bot/constants.py#L278

Reply:
Okay, that's the production host and it should stay that way. Therefore, it's best to leave it set to `redis` within the compose file since it can spin up a redis container. If running without Docker Compose, I presume users are already told they have to manually enable fakeredis?

Reply:
I've added the default back in.
They are, but it's a little buried in the lancebot wiki atm https://pythondiscord.com/pages/contributing/sir-lancebot/sir-lancebot-env-var-reference/
I'm planning on drafting a guide to be added onto the wiki for the docker setup (As it's not currently documented).